lib/Convos/Plugin.pm

```diff
@@ -0,0 +1,39 @@
+package Convos::Plugin;
```

Comment:
Is there really any reason for this..? I think it's cleaner than inheriting directly from Mojolicious::Plugin.